test/e2e/tests/from-import-ui.spec.js

```diff
@@ -191,8 +191,9 @@ describe('MetaMask Import UI', function () {
         await driver.clickElement('.account-menu__icon');
         await driver.clickElement({ text: 'Import account', tag: 'div' });
 
-        // enter private key',
-        await driver.fill('#private-key-box', testPrivateKey1);
+        // enter private key
+        await driver.delay(regularDelayMs);
```

Comment:
Hey @DDDDDanica, you should be able to use `await driver.waitForSelector()` against `#private-key-box` here instead. This will sometimes be faster and more consistent across local and CI machines.

Reply:
Thanks for suggestion, i changed the strategy a bit and using waiting for button to be clickable first. I hope this could be the better solution.